packages/fork-choice/src/protoArray/interface.ts

```diff
@@ -89,6 +89,10 @@ export type ProtoBlock = BlockExtraMeta & {
 
   // Indicate whether block arrives in a timely manner ie. before the 4 second mark
   timeliness: boolean;
+
+  // GLOAS: The followings are from bids. Used for execution payload gossip validation
+  builderIndex?: number;
+  blockHashHex?: RootHex;
```

Comment:
shouldn't this information be part of block input, was there any other use case besides validating the execution payload envelope?

I guess we still need to think about how `BlockInput` looks like for gloas but I'd think we can store it there and just remove block input if all data has been imported and keep it open otherwise

cc @matthewkeil @wemeetagain curious what you think